0012-common-ledger-api/0012-common-ledger-api.md

```diff
@@ -0,0 +1,1201 @@
+# Common Ledger API
+
+The Common Ledger API is a RESTful API served by a ledger (or an adapter), which provides functionality necessary for ILP compatibility. The Common Ledger API provides a single standard API that a ledger can serve in order to ease integration with other Interledger Protocol components and applications, such as the reference ILP Client and ILP Connector. This is not the only way a ledger can become ILP-enabled, but it provides a template that minimizes the integration work necessary for compatibility with ILP software.
+
+## Overview
+
+The Common Ledger API defines the following data types and structures:
+
+- [Transfer resource][]
+- [Account resource][]
```

Comment:
From a blockchain perspective, the account resource belongs to the wallet, not the ledger itself.
Blockchain or append-only ledgers only rely on pubkeys.
I'm not saying we should throw this out of the door, but it seems to me to get this API working for both legacy payment ledgers and blockchain ledgers such as BTC, ETH, BDB, etc there would be the need of a proxy API relaying between the chain and the wallet.

Any thoughts?